docs/examples/embeddings/voyageai.ipynb

```diff
@@ -0,0 +1,84 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/run-llama/llama_index/blob/main/docs/examples/embeddings/voyageai.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Voyage Embeddings"
```

Comment:
Voyage or VoyageAI?

Reply:
Hi @nerdai, we use Voyage Embedding for the model.

Reply:
Regarding the import issue, your suggested solution may not work because the other methods of the class is using the voyageai package. Putting the import in the __init__ function doesn't allow other functions to use the package. Do you have other suggestions?

Reply:
@nerdai I proposed a solution. Please review. Thanks!

Reply:
@thomas0809 I think you only have instance methods and not static or class ones that rely on `voyageai` package. Meaning you'd have to instantiate an object first before using the instance methods that do require this library.